README.md

```diff
@@ -58,6 +58,7 @@ This repo periodically syncs all official Kubeflow components from their respect
 | KServe Models Web App | contrib/kserve/models-web-app | [v0.10.0](https://github.com/kserve/models-web-app/tree/v0.10.0/config) |
 | Kubeflow Pipelines | apps/pipeline/upstream | [2.0.0-alpha.7](https://github.com/kubeflow/pipelines/tree/2.0.0-alpha.7/manifests/kustomize) |
 | Kubeflow Tekton Pipelines | apps/kfp-tekton/upstream | [v1.5.1](https://github.com/kubeflow/kfp-tekton/tree/v1.5.1/manifests/kustomize) |
+| BentoML | contrib/bentoml/bentoml-yatai-stack/default | [v1.7.0](https://github.com/ssheng/manifests/tree/master/contrib/bentoml/bentoml-yatai-stack/default) |
```

Comment:
Can we remove this as well as it's not official component of Kubeflow and is not installed by default

Reply:
Removed.